app/templates/components/crate-list-newest.hbs

```diff
@@ -0,0 +1,12 @@
+<ol>
+  {{#each crates as |crate index|}}
+    <li>
+      {{#link-to 'crate.version' crate.id crate.newest_version class='name' data-test-crate-link=index}}
```

Comment:
So the "Just Updated" list shows a version and links to that version, and the "Most downloaded" lists don't show a version and don't link to a version... I wonder if, for consistency, we should make the "New Crates" list items link to the version because it displays a version? It probably doesn't matter in practice because new crates are only likely to have one version, but what do you think?

Reply:
I wondered about this. It felt weird to link to a specific version when we expect there to be only one. What do you think of not displaying the version number in the "New Crates" list? The meaning of _version number_ isn't very deep when there is only one.

Reply:
Hmmm. Hmmm... the only slight pause I have is that the version number a new crate starts with is a small signal... if it's 0.0.1, it's probably someone's experiment... if it's 1.0.0, it's the launch of something someone's probably been working on for a while and wants to signal that people should use it... is that an interesting enough reason to keep the display of the version number for new crates? I don't know!

Reply:
Yes. I can see your point about version number being a signal for new crates. In addition, having the version number visible for new crates doesn't strike me as detrimental on the crates.io home page. So: my remaining question is just: what do we link to?
- Linking to the specific version aligns with the change in this PR makes for _Just Updates_, making link behavior more consistent.
- Linking to the front page of for the crate seems simpler; it gives a shorter url and a smidge less complexity. If someone sat looking at the crates.io front page for a while, then clicked on a new crate, there is a chance that crate had already published another version. Linking to the crate home would make that newer version more visible.

Reply:
I went with linking to the specific version under *New Crates*. That feels most right to me: consistent with *Just Updated* and I think my stated concern about yet newer versions isn't worth worrying about. The link the user clicks on takes them to what the text says, which, I think, meets the principle of least astonishment.